spec/pluralrules.html

```diff
@@ -172,19 +172,14 @@ <h1>Intl.PluralRules.prototype.resolvedOptions ( )</h1>
         1. Let _pr_ be the *this* value.
         1. Perform ? RequireInternalSlot(_pr_, [[InitializedPluralRules]]).
         1. Let _options_ be OrdinaryObjectCreate(%Object.prototype%).
+        1. Let _pluralCategories_ be a List of Strings containing all possible results of <emu-xref href="#sec-pluralruleselect">PluralRuleSelect</emu-xref> for the selected locale _pr_.[[Locale]].
         1. For each row of <emu-xref href="#table-pluralrules-resolvedoptions-properties"></emu-xref>, except the header row, in table order, do
           1. Let _p_ be the Property value of the current row.
           1. Let _v_ be the value of _pr_'s internal slot whose name is the Internal Slot value of the current row.
+          1. If _p_ is *"roundingIncrement"*, then
+            1. Perform ! CreateDataProperty(_options_, *"pluralCategories"*, CreateArrayFromList(_pluralCategories_)).
```

Comment:
I don't like this piggybacking; instead **"pluralCategories"** should be added to the table with no corresponding Internal Slot and the algorithm should probably follow the pattern of [DateTimeFormat](https://tc39.es/ecma402/#sec-intl.datetimeformat.prototype.resolvedoptions):
```suggestion
        1. For each row of <emu-xref href="#table-pluralrules-resolvedoptions-properties"></emu-xref>, except the header row, in table order, do
          1. Let _p_ be the Property value of the current row.
          1. If _p_ is *"pluralCategories"*, then
            1. Let _pluralCategories_ be a List of Strings containing all possible results of <emu-xref href="#sec-pluralruleselect">PluralRuleSelect</emu-xref> for the selected locale _pr_.[[Locale]].
            1. Let _v_ be CreateArrayFromList(_pluralCategories_).
          1. Else,
            1. Let _v_ be the value of _pr_'s internal slot whose name is the Internal Slot value of the current row.
```

Reply:
Done (similarly)